cura/PrintersModel.py

```diff
@@ -53,9 +53,8 @@ def _update(self) -> None:
         container_stacks = ContainerRegistry.getInstance().findContainerStacks(type = "machine")
 
         for container_stack in container_stacks:
-            connection_type = container_stack.getMetaDataEntry("connection_type")
-            has_remote_connection = connection_type in [ConnectionType.NetworkConnection.value, ConnectionType.CloudConnection.value]
-
+            connection_type = container_stack.getMetaDataEntry("connection_type", 0)
```

Comment:
This has been fixed in 4.0 branch already by defaulting to ConnectionType.NotConnected (which is also 0, but is a bit clearer)